### PR TITLE
Documentation updates for APIs related to webhook changes

### PR DESCRIPTION
**Pull request by robfusion**
--https://github.com/FusionAuth/fusionauth-issues/issues/1812

Documentation updates for APIs related to webhook changes.

### CODE REVIEW

File: site/docs/v1/tech/apis/_webhook-request-body.adoc [hunk 1]
```diff
@@ -46,6 +48,9 @@ The read timeout in milliseconds used when FusionAuth sends events to the Webhoo
 [field]#webhook.sslCertificate# [type]#[String]# [optional]#Optional#::
 An SSL certificate in PEM format that is used to establish the SSL (TLS specifically) connection to the Webhook.
 
+[field]#webhook.tenantIds# [type]#[Array<UUID>]# [optional]#Optional# [since]#Available since 1.37.0#::
```
Comment (spwitt) [inline site/docs/v1/tech/apis/_webhook-request-body.adoc:51]: Should webhook request/response descriptions for `tenantIds` be updated to reference the `applicationIds` field to help people on older versions find the correct field?

File: site/docs/v1/tech/core-concepts/applications.adoc [hunk 2]
```diff
@@ -505,17 +505,17 @@ For example, if a password needs to be stored in an external configuration and t
 
 === Webhooks
 
+[WARNING.warning]
+====
+This feature is removed as of version 1.37.0.
+====
+
 The Webhooks tab allows you to select one or more webhooks to be used for this Application. In this example screenshot either no webhooks have been configured, or no application specific webhooks are configured.
```
Comment (spwitt) [inline site/docs/v1/tech/core-concepts/applications.adoc:513]: Should a version of this section be added to the Tenant core concepts page? I'm not sure whether there was additional webhook configuration added at the tenant level.

Comment (robfusion) [reply site/docs/v1/tech/core-concepts/applications.adoc:513]: In the upcoming PR

File: site/docs/v1/tech/events-webhooks/writing-a-webhook.adoc [hunk 3]
```diff
@@ -41,6 +42,16 @@ If you have multiple tenants listening for the same event, they will all receive
 
 include::docs/v1/tech/events-webhooks/_application-webhooks-warning.adoc[]
 
+=== Tenant Scoped Events
+
+As of version 1.37.0, all events can be tenant scoped except instance events:
```
Comment (robotdan) [inline site/docs/v1/tech/events-webhooks/writing-a-webhook.adoc:47]: In the admin UI we are calling these system events. We could use the same wording here, or explain what an instance event is.

**Pull request merged**